Libraries/oneDPL/gamma-correction/gamma-correction.vcxproj

```diff
@@ -114,7 +114,7 @@
       <ConformanceMode>true</ConformanceMode>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <SYCLWarningLevel>DisableAllWarnings</SYCLWarningLevel>
-      <AdditionalIncludeDirectories>$(ONEAPI_ROOT)\tbb\latest\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(ONEAPI_ROOT)tbb\latest\include;$(ONEAPI_ROOT)dpl\latest\windows\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
```

Comment:
@capatober -- you're relying on $(ONEAPI_ROOT) to end in a '\\' character. I don't think that's a safe assumption. If setvars.bat is run before starting Visual Studio that variable gets overwritten and, in the current iteration, does not have a trailing backslash. 

I would assume this path list in Visual Studio does not care if it finds a double-backslash. I know that Windows does not care, since it will happen in cases like this. This would have to be tested in Visual Studio to ensure that a double-backslash embedded within a path gets treated like a single backslash, just like it does in general, in Windows.

Reply:
@xmnboy, thanks for the feedback.
Am I right that you suggest to return `\` back?

Reply:
The following (in the `vcxproj` file `AdditionalIncludeDirectories` entry):
```
$(ONEAPI_ROOT)dpl\latest\windows\include
```
Should be implemented as:
```
$(ONEAPI_ROOT)\dpl\latest\windows\include
```

Reply:
Fixed by #348